backend/api/src/test/kotlin/software/shonk/adapters/incoming/ShorkInterpreterControllerV1IT.kt

```diff
@@ -57,4 +54,29 @@ class ShorkInterpreterControllerV1IT() : AbstractControllerTest() {
         assertEquals(HttpStatusCode.BadRequest, result.status)
         assert(result.bodyAsText().contains("No player with that name in the lobby"))
     }
+
+    @Test
+    fun `test get lobby status with invalid ID`() = runTest {
+        val result = client.get("/api/v1/lobby/status/1")
+        assertEquals(HttpStatusCode.BadRequest, result.status)
+        assert(result.bodyAsText().contains("No lobby with that id"))
+    }
+
+    @Test
+    fun `test get lobby status with valid (default) ID`() = runTest {
+        val result = client.get("/api/v1/lobby/status/0")
+        val expectedStatus =
+            """
+        {
+            "playerASubmitted": false,
+            "playerBSubmitted": false,
+            "gameState": "NOT_STARTED",
+            "result": {
+                "winner": "UNDECIDED"
+            }
+        }
+        """
```

Comment:
This should be done via parsing, not via string equality checking.
This was already changed in the past and is already in main: https://github.com/corewar-teamprojekt/corewar/pull/160/commits/3f4854e077cf1290b4552709b9bf73016507c60b

Reply:
I was actually using it in the beginning but I wasn't sure if that was ok, but apparently it was the right decision, I'm going to have to revert back to it